docs/contribute/test/testing.md

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 1
+---
+
+# Testing
+
+This module will teach you about writing tests for C++ Code for adding test suites for new plugins or modules for WasmEdge Software. This is an important aspect if you want to submit plugins to the WasmEdge community and we would require you to write test suites to cater to the plugin functionalities.
+
+## Writing Tests for C++ Code Targeting WebAssembly
```

Comment:
You also mean compiling the C++ code to the Wasm file here.

Reply:
Will update it